### ISSUE TITLE
Proxmox vm Hub9 30 breaks during install of components

### ISSUE THREAD

**Issue by Allexz1** [nextcloud/vm#2699]
### Steps To Reproduce

Following the installer script breaks the installer in a few places.

If choosing to install FullTextSearch during the installationphase the script breaks just before starting the docker. (sadly no more info here)

If choosing to install Talk record during the installationphase the script breaks just before starting the docker due to a dependency issue.
the script tries to apt install netcat, but the [docker forum](https://forums.docker.com/t/package-netcat-has-no-installation-candidate-how-to-fix-this/136541) shows that netcat-traditional should be used instead. I will be trying this next as a post install.

### Expected Result

Choosing options in the script should not break the entire installer :) 

### Actual Result

Script breakage, vm removal, restart.

### Additional Context

It would be desirable to be able to go backwards during the automated installation, if it was the case i could have stepped backwards and not broken the script. In both cases i had to entirely remove the vm and start from scratch.

### Build Version

latest

### Environment

By downloading the VM

### Environment Details

Proxmox hypervisor.

**Comment by enoch85** [comment]
> If choosing to install FullTextSearch during the installationphase the script breaks just before starting the docker. (sadly no more info here)

We have several pop ups during installation that we _don't recommend_ installing apps during first setup. Wait with that until after the installation is done - then you'll get the latest fixes of all the app scripts.

I've replaced `netcat` now as per your recomendation, thanks! 👍

**Comment by enoch85** [comment]
> It would be desirable to be able to go backwards during the automated installation, if it was the case i could have stepped backwards and not broken the script. In both cases i had to entirely remove the vm and start from scratch.

Yeah, in a perfect world. :) Starting from scratch on the _first setup_ isn't so hard and is right now the only way I'm afraid.

I've investigated this now on a fresh system and can't find any errors. Closing.

Btw, thanks for posting your issue! :)
